runtime/lua/vim/treesitter/_fold.lua

```diff
@@ -392,7 +394,7 @@ function M.foldexpr(lnum)
   lnum = lnum or vim.v.lnum
   local bufnr = api.nvim_get_current_buf()
 
-  local parser = vim.F.npcall(ts.get_parser, bufnr)
+  local parser = ts._get_parser(bufnr)
```

Comment:
(off topic) perhaps everywhere we use `npcall` should be reevaluated...

Reply:
That is the hope, yes (one API at a time...)